core-services/beneficiary-idgen/src/main/resources/db/migrate.sh

```
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true -outOfOrder=true migrate
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)